python/sbp/settings.py

```diff
@@ -300,7 +300,7 @@ def to_json_dict(self):
 class MsgSettingsReadByIndexReq(SBP):
   """SBP class for message MSG_SETTINGS_READ_BY_INDEX_REQ (0x00A2).
 
-  You can have MSG_SETTINGS_READ_BY_INDEX_REQ inherent its fields directly
+  You can have MSG_SETTINGS_READ_BY_INDEX_REQ inherit its fields directly
```

Comment:
Eh, yeah, it probably would have made more sense to keep the generated python stuff with this comment spelling correction into a separate commit.